packages/shared/src/hooks/usePostById.ts

```diff
@@ -129,7 +131,9 @@ const usePostById = ({ id, options = {} }: UsePostByIdProps): UsePostById => {
     staleTime: StaleTime.Default,
     enabled: !!id && tokenRefreshed,
   });
-  const post = postById || (options?.initialData as PostData);
+  const post = isPlus
+    ? postById
+    : postById || (options?.initialData as PostData);
```

Comment:
Not ideal, but doesn't seem to cause any noticeable delay

Reply:
Should we do this though, considering it wasn't so bad on preview?

Reply:
This was the preview

Reply:
but it basically kills the SSR if you are on plus, I don't think we should do this

If  we really want to avoid that shift we should show some kind of loader only for title part but the rest of the page should use SSR data

Reply:
Since we have it for language already also think maybe leaving it out for now is best.